script_runner.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, job_id, shippable_adapter, \
         self.console_buffer = []
         self.console_buffer_lock = threading.Lock()
         self.continue_trigger_flush_console_output = True
-        self.max_consoles_size_in_bytes = 64 * 1024 * 1024  # 64 MB
+        self.max_consoles_size_in_bytes = self.config['MAX_CONSOLES_SIZE_MB'] * 1024 * 1024
```

Comment:
why not convert this to bytes upfront in config.py. we shouldn't be tweaking this value once its processed